array/array.js

```diff
@@ -7,3 +7,78 @@
  * How do these operations compare to that of a linked list?
  * How does the time complexity of insertion and deletion compare to that of a linked list?
  */
+
+class Array {
+  constructor() {
+    this.head = null;
+    this.tail = null;
+    this.length = 0;
+  }
+  push(element) {
+    const newNode = {
```

Comment:
Great idea on using a linked list here. I've seen this done in this way before. There are more concise and simpler solutions out there, and i recommend looking at those too! But I LOVE this idea of piggy backing off your previous knowledge.